docs/toml.md

```diff
@@ -141,7 +141,9 @@
 email = "test@traefik.io"
 
 # File used for certificates storage.
-# WARNING, if you use Traefik in Docker, don't forget to mount this file as a volume.
+# WARNING, if you use Traefik in Docker, don't forget to mount the folder containing this file as a volume.
+# storageFile = "/etc/traefik/acme/acme.json"
+# $ docker run -v "/my/host/acme:/etc/traefik/acme" traefik
```

Comment:
In the documentation I think we should specify both 😇. Then the user has the choice of which one to use 😝.